src/actions/index.test.js

```diff
@@ -0,0 +1,50 @@
+import { NETWORK_KINDS } from '../constants/network';
+import {
+  NEW_NETWORK,
+  LOAD_NETWORK,
+  CHANGE_NETWORK_PROPERTY,
+  ADD_NODE,
+  newNetwork,
+  loadNetwork,
+  changeNetworkProperty,
+  addNode,
+} from './index';
+
+describe('index file tests', () => {
+  it('newNetwork ', () => {
+    expect(newNetwork).toBeInstanceOf(Function);
+    newNetwork((ret) => {
+      expect(ret.type).toEqual(NEW_NETWORK);
+      expect(ret.kind).toEqual(NETWORK_KINDS.BN);
```

Comment:
Actually, this snippet of code is never triggered and so, this test will never fail.
And even if it was would be better to check the hole `ret` to make sure nothing changes, EX.:

This test (if works) would never fail if `newNetwork` added a new prop:
```js
{
  type: NEW_NETWORK,  // checked by the test
  kind: NETWORK_KINDS.BN,  // checked by the test
  newProp: 'any-value',  // NEVER checked by the test
}
```

One tip about making tests, it's a good practice always make it fail at least once just to make sure your test is testing properly. Ex.:
```js
expect(ret.type).toEqual('pão de bataaaata'); // this should fail
```
;D